src/types-and-values/strings.md

```diff
@@ -23,34 +23,33 @@ fn main() {
     //println!("{:?}", &sentence[12..13]);
 }
 ```
-
 <details>
-
 This slide introduces strings. Everything here will be covered in more depth
 later, but this is enough for subsequent slides and exercises to use strings.
-
 - Invalid UTF-8 in a string is UB, and this not allowed in safe Rust.
-
 - `String` is a user-defined type with a constructor (`::new()`) and methods
   like `s.push_str(..)`.
-
 - The `&` in `&str` indicates that this is a reference. We will cover references
   later, so for now just think of `&str` as a unit meaning "a read-only string".
-
+  ```rust,editable
+  let greeting = "Greetings" // -> This type of string declaration is by default comes under &str type.
+  let mut sentence:String = greeting; // Output : Error because `greeting` is &str type. 
+  // You can convert `&str` to `String` by `to_string()` function.
+  // i.e 
+  let mut sentence = greeting.to_string()
+  ```
```

Comment:
I think this is too early in the course to talk about `to_string` -- it would be better handled later, in the `String` slide in the standard-library types segment.

Reply:
Hello should I add this in String slide in standard-library types segment.